### DISCUSSION TITLE
How to use different matrix sizes ?

### DISCUSSION THREAD

**Discussion by Big-Brummer** [pixelit-project/PixelIt#288] [Q&A] [answered]
Dear all, I found this great project and I love it.
I wounder if it is possible to use different matrix sizes .

Is it possible to use only a single 8x8 matrix?
Is it possible to change to 10x20 matrix ?

many thanks for answers.

**Comment by foorschtbar** [comment, marked as answer]
> Is it possible to use only a single 8x8 matrix? Is it possible to change to 10x20 matrix ?

Hi, unfortunately the resolution is fixed to 8x32 pixel - but: it is possible to change the code in the right places to fix the problem yourself.

**Comment by mxnx88** [comment]
Hi, where can I find the right place in the code? In the PixelIt.ino?

many thanks for the project

**Comment by Big-Brummer** [comment]
I did not solve ist
